CameraTutorial/Program.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace LearnOpenGL_TK
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            using (Game game = new Game(800, 600, "LearnOpenGL"))
```

Comment:
I would vote for "LearnOpenTK - Camera Tutorial" as the title.

Reply:
I should probably give the other tutorials a more descriptive window title, will do that after this is finished.

Reply:
I also cannot do this with jetbrains rider as both ' ' and '-' are characters i cannot call my projets

Reply:
I'm not on about the project, I'm on about the window title used in this constructor.

Reply:
Well in that case i think imo this should be done in a seperate PR with a slight refactor of all the tutorials

Reply:
You should fix this now in this PR. I'll do one massive code review once the whole of LearnOpenTK is done.